docs/sources/configure-client/language-sdks/java.md

```diff
@@ -29,21 +29,26 @@ You can start Pyroscope either from your apps's Java code or attach it as javaag
 ## Start Pyroscope from app's Java code
 First, add Pyroscope dependency
 
-### Maven
-```xml
+{{< code >}}
```

Comment:
It doesn't seem to like `<code>`
- https://github.com/grafana/pyroscope/actions/runs/6177786541/job/16769690996?pr=2399#step:3:80

Can these tags be removed?

Reply:
they should be fine we need to update our pipeline to accept them I guess.. @jdbaldry is going to help with this when he gets time

Reply:
I have updated the `grafana/docs-base:latest` image so that it now includes the `code` shortcode. You should be able to restart the workflow and have it build correctly now.